samples/Samples.SqlServer/Properties/launchSettings.json

```diff
@@ -11,6 +11,7 @@
         "CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
         "CORECLR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
 
+        "DD_DOTNET_TRACER_HOME": "$(ProjectDir)$(OutputPath)profiler-lib",
         "DD_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json"
```

Comment:
Should we drop `DD_INTEGRATIONS` and figure it out from `DD_DOTNET_TRACER_HOME` instead? (Can be separate PR)

Reply:
Is there anywhere where we might need the flexibility of an integrations file outside of the profiler directory? Perhaps we could default to the DD_DOTNET_TRACER_HOME directory and use DD_INTEGRATIONS as a sort of override?

Reply:
> default to the `DD_DOTNET_TRACER_HOME` directory and use `DD_INTEGRATIONS` as a sort of override

I like this idea. The latter also lets us define multiple json files. We should probably also rename that to `DD_DOTNET_INTEGRATIONS` at some point.

Let's leave as-is for this PR and come back to that later.